lib/coophub/backends/backends.ex

```diff
@@ -32,6 +32,8 @@ defmodule Coophub.Backends do
   defp get_backend_module!("github"), do: Backends.Github
   defp get_backend_module!("gitlab"), do: Backends.GitlabCom
   defp get_backend_module!("git.coop"), do: Backends.GitCoop
+  defp get_backend_module!("0xacab"), do: Backends.HexAcab
+  defp get_backend_module!("hexacab"), do: Backends.HexAcab
```

Comment:
```suggestion

```
Please remove this line if not used

Reply:
I see, nice, yes, let's remove this line :)

Reply:
Done!